src/context/SWR.tsx

```diff
@@ -83,6 +83,9 @@ const SwrConfigProvider = ({ children }: BaseComponentProps) => {
                             Boolean(supabaseClient.auth.user());
 
                         if (localUserInvalid || tokenHasIssues(error.message)) {
+                            logRocketEvent(CustomEvents.AUTH_SIGNOUT, {
+                                trigger: 'swr',
+                            });
```

Comment:
Added some eventing to keep an eye on when users get signed out. This even does not get trigger on every single call to `auth.signOut` but the main ones that auto log a user out due to an issue.